misc/nvme-bootstrap/kubernetes/daemonset.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Do we also need those yaml files as well? 

We have this as an example in Terraform already: https://github.com/MaterializeInc/terraform-aws-materialize/pull/41/files#diff-17bfdd719dbe7e71d2a1611e4312e71e140ca1c05f2ba6cddf62b7ec4d1c38c2R113